app/models/user.rb

```diff
@@ -1,12 +1,20 @@
 class User < ApplicationRecord
-
   # Connects this user object to Blacklights Bookmarks.
   include Blacklight::User
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
-  devise :database_authenticatable, :registerable,
```

Comment:
Just double checking that you intended to remove `registerable`.

Reply:
we don't want any users to register. we need authenticated users to run the import/index functions, but there will be no other registered users